### PR TITLE
Adding additional attributes to support intelligent services Schema

### PR DESCRIPTION
**Pull request by prabhum2**
This is work in progress - DONT MERGE

As a part of intelligent service schema onboarding added more values to channel types .

### CODE REVIEW

File: schemas/context/normalized-experianceevent.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    "https://ns.adobe.com/xdm/context/experienceevent-media",
+    "https://ns.adobe.com/xdm/context/experienceevent-profile-stitch",
+    "https://ns.adobe.com/xdm/context/experienceevent-search",
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-technical-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-web"],
+  "definitions": {
+    "normalized-experienceevent": {
+      "properties": {
+          "xdm:eventType": {
```
Comment (cmathis) [inline schemas/context/normalized-experianceevent.json]: A context/experienceevent (which this mixin is meta:intendedToExtend) already include the data/time-series schema which has an eventType field already.

**Comment by cmathis** [comment]
It appears that this normalized-experienceevent is a new mixin that simply wraps all other available standard mixins. Basically takes us back to the original uber experienceevent we had before. 

Who is using this schema, and how will the Dataset it's associated with be populated?

**Comment by forpankil** [comment]
Why aren't we re-using existing mix-ins like "environment","search", "web", "application","commerce","marketing" ?

**Comment by cmathis** [comment]
@prabhum2  - I do not understand what you are doing here. You appear to be duplicating all the existing objects but giving them a new $id that includes "std" in the value. Is that the purpose?

**Pull request merged**